test_xdb/models/under_test/clone_schema_sequences_metadata_equality_test.sql

```diff
@@ -0,0 +1,129 @@
+--depends_on: {{ ref('clone_schema_tables_metadata_equality_test') }}
```

Comment:
This model shows us lists of metadata fields with deltas between original and copied sequences in test schemas.